downloader/src/main/kotlin/vcs/Git.kt

```diff
@@ -121,7 +122,7 @@ class Git : GitBase() {
         }
 
         // Fall back to fetching everything.
-        return try {
+        return succeeds({
```

Comment:
Using the `succeeds` help function instead of catching a specific exception can hide bugs, because the generic `Throwable` is catched in this case. Also IMO the variant with two lambda arguments makes it harder to read the code, because it's unintuitive. I would suggest to make the helper function internal and to only use it in the simple cases of the extension functions in `Extensions.kt`.

Reply:
Solved almost like that: I left the function public in case someone still wants to use it form the outside.